### PR TITLE
CNV example output and some source-y updates

### PR DESCRIPTION
**Pull request by aburdenthehand**
- Adding Example output heading to example output, and separating output from commands where applicable. 
- In these files I've added '[source,terminal]' tags to terminal commands (but have not undertaken this for all modules). 
- Also fixing some module metadata and adding/correcting module IDs.

### CODE REVIEW

**Comment by ousleyp** [comment]
This should go into enterprise-4.5 as well, right?

File: modules/virt-importing-vm-cli.adoc [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // Module included in the following assemblies:
+//
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
-[id='virt-importing-vm-cli_{context}']
+
+[id="virt-importing-vm-cli_{context}"]
```
Comment (ousleyp) [inline modules/virt-importing-vm-cli.adoc:5]: Seriously good catches on these module IDs. This is the kind of thing I miss when doing reviews. 😱

File: modules/virt-troubleshooting-incorrect-policy-config.adoc [hunk 2]
```diff
@@ -66,12 +72,15 @@ However the Policy status alone does not indicate if it failed on all nodes or a
...
 $ oc get nnce
 ----
 +
 The output shows that the Policy failed on all nodes:
 +
+.Example output
+[source,terminal]
 ----
 NAME                                   STATUS
 utput
```
Comment (ousleyp) [inline modules/virt-troubleshooting-incorrect-policy-config.adoc:86]: utput?

this clearly wasn't part of this PR, but it seems like a typo

File: modules/virt-viewing-virtual-machine-logs-cli.adoc [hunk 3]
```diff
@@ -11,6 +11,7 @@ Get virtual machine logs from the virtual machine launcher Pod.
 
 * User the following command:
```
Comment (ousleyp) [inline modules/virt-viewing-virtual-machine-logs-cli.adoc:12]: "User the following command:"

Once again, this clearly isn't part of this PR but I can't unsee it

**Review by ousleyp — APPROVED** [review]
LGTM. I noticed a couple of unrelated typos, so I called them out. It seems as good a time as any to fix them.

File: modules/virt-troubleshooting-incorrect-policy-config.adoc (hunk 2 above)
Comment (aburdenthehand) [reply modules/virt-troubleshooting-incorrect-policy-config.adoc:86]: Niiiice catch.
That shouldn't be there at all.
Gone

**Pull request merged**